src/dashboard/pages/DashboardPage.vue

```diff
@@ -47,13 +47,15 @@ div
     v-container(grid-list-md)
       // Favorite creators
       v-layout(row)
-        v-flex(xs12).mb-6
-          div.display-1 Your favorite creators
-        v-flex(grow)
+        v-flex
+          span.display-1 Your favorite creators
+          span.ml-3.caption(v-show="daysSinceDonation !== null")
+            | You last donated {{daysSinceDonation}} days ago
```

Comment:
Maybe find different placement for the text? Inside donation summary?

Reply:
I want it to be visible from the moment you open the dashboard. DonationSummary shouldn't do even more things, if anything it should do less.